website/content/docs/install/index.mdx

```diff
@@ -230,6 +230,18 @@ net.bridge.bridge-nf-call-iptables = 1
 
 </CodeBlockConfig>
 
+<h3>Install `consul-cni` for Consul service mesh</h3>
+
+If the `transparent_proxy` block is used for Consul Service Mesh, then the 
+`consul-cni` CNI Plugin must be installed on each client node to redirect inbound 
+and outbound traffic for services to the Envoy proxy. The following commands
+assume that reference CNI plugins have already been installed. 
+
+```shell-session
+$ curl -L -o consul-cni.zip "https://releases.hashicorp.com/consul-cni/1.5.0/consul-cni-1.5.0_linux_$( [ $(uname -m) = aarch64 ] && echo arm64 || echo amd64)".zip && \
+  unzip consul-cni.zip -d /opt/cni/bin 
```

Comment:
This architecture We should break these huge long lines:

```suggestion
$ curl -L -o consul-cni.zip \
  "https://releases.hashicorp.com/consul-cni/1.5.0/consul-cni-1.5.0_linux_$( [ $(uname -m) = aarch64 ] && echo arm64 || echo amd64)".zip \
  && unzip consul-cni.zip -d /opt/cni/bin 
```

Or potentially even pull the arch subshell command out to a variable to make it nicer to read?

Reply:
Thanks, done.